pkg/util/tracing/tracingpb/recorded_span.proto

```diff
@@ -103,17 +108,27 @@ message RecordedSpan {
   // DeprecatedInternalStructured only stores the Payloads.
   repeated StructuredRecord structured_records = 14 [(gogoproto.nullable) = false];
 
+  // Next ID: 19
+
   reserved 5,10,11;
 }
 
+message CompositeTag {
```

Comment:
This could be a `oneof: value, tags`, but it's kind of a pain to nest a map in a `oneof`.